packages/block-library/src/social-links/style.scss

```diff
@@ -91,8 +91,6 @@
 // This needs specificity because themes usually override it with things like .widget-area a.
 .wp-block-social-links .wp-block-social-link.wp-social-link {
 	display: inline-block;
-	margin: 0;
-	padding: 0;
```

Comment:
What's the thinking behind removing these resets?

My understanding was they should be tweaked to lower the specificity so global styles could be applied. Was there an unexpected issue when doing so?

Reply:
Should we move these style to social link block? Global style would be able to override that style. Another solution is we can add selector in social link block for spacing styles.

Reply:
I think this might need some further thought. 

The Social Link block contains a clickable button or link depending on whether the user is in the editor or frontend. 

It would make sense to me that any padding added for a link would be on that inner button or link so it is taken into account for the clickable area. The same could be said for the border too.

This would likely mean, leaving the styles here untouched and skipping serialization for the border and padding support. Then manually applying the support styles on the appropriate inner element.